packages/main/src/ToolbarItemTemplate.tsx

```diff
@@ -0,0 +1,9 @@
+import type ToolbarItem from "./ToolbarItem.js";
+
+export default function ToolbarItemTemplate(this: ToolbarItem) {
+	return (
+		<div onClick={this.onClick}>
```

Comment:
Once we have a sample with different types of components we should test if this click only event handling is enough to manage the closing of the popover. If not we should extend it or think of another mehanism.